modules/api/src/main/java/org/apache/ignite/table/manager/IgniteTables.java

```diff
@@ -71,7 +74,7 @@
      * @param tableInitChange Table changer.
      * @return Existing or newly created table.
      */
-    Table getOrCreateTable(String name, Consumer<TableChange> tableInitChange);
+    Table createTableIfNotExists(String name, Consumer<TableChange> tableInitChange);
```

Comment:
Please, here and for `createTableIfNotExistsAsync`, add note, that will be returned table with given name, but its configuration will not validate through configuration and `tableInitChange.`